{{cookiecutter.plugin_name}}/setup.cfg

```
@@ -0,0 +1,45 @@
+[metadata]
+name = {{cookiecutter.plugin_name}}
+author = {{cookiecutter.full_name}}
+author_email = {{cookiecutter.email}}
+license = {{cookiecutter.license}}
+url = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}
+description = {{cookiecutter.short_description}}
+long_description = file: README.md
+long_description_content_type = text/markdown
+classifiers =
+    Development Status :: 4 - Beta
```

let's change this now too:

```suggestion
    Development Status :: 2 - Pre-Alpha
```